action.yml

```diff
@@ -62,18 +62,18 @@ runs:
       uses: helm/kind-action@0025e74a8c7512023d06dc019c617aa3cf561fde # v1.10.0
       with:
         version: v0.21.0
-        kubectl_version: v1.28.6
+        kubectl_version: v1.29.1
         cluster_name: solo-e2e
         wait: 120s
 
     - name: Install Solo
       shell: bash
-      run: npm install -g @hashgraph/solo@0.28.1
+      run: npm install -g @hashgraph/solo@0.31.4
 
     - name: Deploy Solo Test Network
       shell: bash
       run: |
-        solo init -i node0 -t ${{ inputs.hederaVersion }} -n solo-test --key-format pem --profile local
+        solo init -i node0 -t v0.54.0 -n solo-test --profile local
```

Comment:
Why have you removed the variable?

Reply:
Aahhh sorry. I was testing and forgot to revert